lib/platform/gitlab/index.ts

```diff
@@ -59,7 +60,11 @@ const defaults = {
   endpoint: 'https://gitlab.com/api/v4/',
 };
 
+const DRAFT_PREFIX = 'Draft: ';
+const DRAFT_PREFIX_DEPRECATED = 'WIP: ';
+
 let authorId: number;
+let draftPrefix: string;
```

Comment:
```suggestion
let draftPrefix = '';
```

This is needed for now to fall back gracefully for the app